### ISSUE TITLE
Add support for switches on inputs

### ISSUE THREAD

**Issue by jonoxer** [SuperHouse/LSC#9]
The current firmware assumes momentary buttons on inputs. It should be possible to configure any arbitrary input as being a switch instead of a button, so that it acts on every transition instead of just when the contact closes. Wall plates could then be assembled using regular mains switches so that they appear unmodified, and behave as expected.

**Comment by sumnerboy12** [comment]
This has been implemented in https://github.com/sumnerboy12/LSC/pull/4 (of my fork). I have renamed the sketch since this now supports more than just light switch events, it will handle any binary input - i.e. BUTTON, SWITCH or CONTACT inputs, reporting appropriate events to MQTT.

I would be happy to create a PR to your repo if you think this has value?

NOTE: I have removed the OLED code for now, as I was trying to conserve space, plus I am unable to test since my OLED module appears to be busted.

Thanks to @moinmoin-sh for his help with developing this new firmeware!
